src/helpers/environment.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+function is_git_repository {
+  if git rev-parse > /dev/null 2>&1; then
+    echo true
+  else
+    echo false
+  fi
+}
+
+function ensure_git_repository {
+  if [ "$(is_git_repository)" == false ]; then
+    echo "This is not a git repository."
+    exit_with_error
+  fi
+}
+
+
+# Bypass the environment checks
+if [[ $@ =~ --bypass-environment-checks ]]; then
+  return 0
+fi
```

Comment:
I think this check should go in helpers.sh as well. That way don't need to pass this through


```sh
if [[ ! $@ =~ --bypass-environment-checks ]]; then
  ensure_git_repository
fi
```